### DISCUSSION TITLE
FS22 large maps marking fields for creation of field catch line boundaries using the ADEditor

### DISCUSSION THREAD

**Discussion by Torqueitup** [KillBait/FS19_AutoDrive_MapImages#36] [Q&A]
Hi, I'm going through the process of applying field boundary catch lines in a couple of large maps - ElmCreekbyStevie is one of them.
I'd like to know quick ways of accurately setting those boundaries by using the ADEditor.  I can drive them but that takes a lot of time and patience.  I'd like somehow to mark the corners or selected waypoints on a map boundary, then using the ADEditor, create the bi-directional boundary.  Has anyone come up with a surefire solution to this?

**Comment by yuccasu** [comment]
I would try to export field boundaries from some file and merge the coordinates to an autodrive configuration. Does anyone know in which file(s) field boundaries are stored? Does this change for different versions FS19/22/25?
